Content.Shared/CCVar/CCVars.cs

```diff
@@ -299,6 +299,19 @@ public sealed class CCVars : CVars
         public static readonly CVarDef<int> NukeopsPlayersPerOp =
             CVarDef.Create("nukeops.players_per_op", 5);
 
+        /*
+         * Revolutionary
+         */
+        
+        public static readonly CVarDef<int> RevolutionaryMinPlayers =
+            CVarDef.Create("revolutionary.min_players", 15);
+
+        public static readonly CVarDef<int> MaxRevolutionaryHeads =
+            CVarDef.Create("revolutionary.max_heads", 3);
+
+        public static readonly CVarDef<int> HeadsPerRevolutionaryHead =
+            CVarDef.Create("revolutionary.heads_per_revo_head", 1);
+
```

Comment:
These should be on the gamerule not as cvars.

Reply:
Nukeops, zombies, pirates, traitor, etc. do it exactly as I've done it here and use the values almost exactly as I've used them

Reply:
They do and they were coded before we had gamerule configurations and need updating.